server/game/core/ongoingEffect/OngoingEffectEngine.ts

```diff
@@ -63,18 +63,19 @@ export class OngoingEffectEngine {
             const context = effect.context;
             const targets = effect.targets;
             return {
-                title: context.source.name + '\'s effect' + (targets.length === 1 ? ' on ' + targets[0].name : ''),
+                title: context.source.title + '\'s effect' + (targets.length === 1 ? ' on ' + targets[0].name : ''),
                 handler: () => {
-                    properties.gameAction.setDefaultTarget(() => targets);
-                    if (properties.message && properties.gameAction.hasLegalTarget(context)) {
+                    // TODO The below line was setting the target to the card originating the effect, even for player effects
```

Comment:
Still true? I assumed this was fixed by switching to a player target effect

Reply:
@AMMayberry1 So it does still set the default target to Zorii Bliss but the DiscardFromHand system doesn't use it, so I'm curious how this would work with a card system vs. a player system

Reply:
Ping me on this in chat, got a few questions. Specifically wondering how `effect.targets` is getting set, seems like there's a bug there if it's presenting a card target default for a player target system